models/park_natural_attraction.py

```diff
@@ -0,0 +1,9 @@
+class Park_Natural_Attraction():
```

Comment:
Underscore in class names is non-conventional. It is recommended to make them Pascal-cased - `ParkNaturalAttraction`